network/meson.build

```diff
@@ -6,4 +6,7 @@ if conf.get('ENABLE_NETWORKD', 0) == 1
                '80-container-vz.network',
                '99-default.link',
                install_dir : networkdir)
+
+  meson.add_install_script('sh', '-c',
+                           mkdir_p.format(sysconfdir + '/systemd/network'))
```

Comment:
```$(prefix)/lib/network``` is missing.

I wonder whether we should simply always create both dirs manually, i.e. the one in /etc and /lib.
While I read you reasoning for not doing so in the commit message, once we stop installing files into one of those directories, we would stop creating it as a side-effect which would be easy to miss.
Always creating both would avoid that.